src/templates/assets/javascripts/integrations/version/correspondingPage.ts

```diff
@@ -0,0 +1,37 @@
+// This is a separate file so that `mocha` can load it without needing
+// DOM shims.
+
+import { Sitemap } from "../sitemap"
+
+type CorrespondingURLParams = {
+  selectedVersionSitemap: Sitemap
+  selectedVersionBaseURL: URL
+  currentLocation: URL
+  currentBaseURL: string
+}
+
+/**
+ * Choose a URL to navigate to when the user chooses a version in the version
+ * selector.
+ *
+ * @param selectedVersionSitemap - as obtained by fetchSitemap from `${selectedVersionBaseURL}/sitemap.xml`
+ * @param selectedVersionBaseURL - usually `${currentBaseURL}/../selectedVersion`
```

Comment:
I am not sure what to do about the eslint warning here, it seems to be a bug in the jsdoc linter.

One possibility is to silence the warning (not sure how).

Another is to have a single `params` argument and move the documentation to the type.

I do like having an object as the parameter, as it makes the tests much more readable.